config/locales/en/course/assessment/assessments.yml

```diff
@@ -27,6 +27,14 @@ en:
           end_at: :'course.assessment.assessments.index.end_at'
           base_exp: 'Base Experience Points'
           bonus_exp: 'Bonus Experience Points'
+          skippable: 'Skip Steps'
+          allow_partial_submission: 'Finalise Anytime'
```

Comment:
`Allow submission with incorrect answers`

For consistency with the assessment edit page.